lib/sevak/core.rb

```diff
@@ -1,15 +1,23 @@
 module Sevak
   module Core
     def connection
-      return @conn if @conn
-      @conn ||= Bunny.new(
-        host: config.host,
-        port: config.port,
-        username: config.user,
-        password: config.password)
-
-      @conn.start
-      @conn
+      begin
+        return @conn if @conn
+        host = config.host.kind_of?(Array) ? config.host : [config.host]
+        @conn ||= Bunny.new(
+          hosts: host,
+          port: config.port,
+          username: config.user,
+          password: config.password)
+        @conn.start
+        @conn
+      rescue Bunny::TCPConnectionFailedForAllHosts => e
+        sleep(0.0001)
```

Comment:
Isn't this sleep time too less? In case if a node is down this will retry 1000 times/s

Reply:
I think the sleep time can be increased This 0.1 ms is too very less.